src/webview.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include "matoya.h"
+
+#define JAVASCRIPT_EVENT_DISPATCH "window.dispatchEvent(new CustomEvent('%s', { detail: %s } ));"
+
+struct webview_common {
+	bool debug;
+	bool hidden;
+	char *html;
+	void *opaque;
+	MTY_EventFunc event;
+};
+
+struct webview *mty_webview_create(void *handle, const char *html, bool debug, MTY_EventFunc event, void *opaque);
+void mty_webview_destroy(struct webview **webview);
+void mty_webview_show(struct webview *ctx, bool show);
+bool mty_webview_is_visible(struct webview *ctx);
+void mty_webview_event(struct webview *ctx, const char *name, const char *message);
+void mty_webview_resize(struct webview *ctx);
+
+#if defined(_WIN32)
+bool mty_webview_has_focus(struct webview *ctx);
+#endif
```

Comment:
It would be better to take this out of this platform-independent header and move it to a windows-specific header in the windows dir called `webvieww.h` or `webview-win.h` or w/e and then define + include that in `appw.c` and `webvieww.c`. 

The matoya philosophy is to avoid platform-specific defines as much as possible.